roles/gearman/meta/main.yml

```diff
@@ -0,0 +1,18 @@
+galaxy_info:
```

Comment:
this file does not make sense under the collection either

Reply:
file dropped